src/pages/404.js

```diff
@@ -56,40 +102,56 @@ const NotFoundPage = () => (
         footerContactUsProfile: { id },
         title,
         copy: { copy },
-        ctaLink,
-        ctaCopy
+        linksTitle
       } = content
       return (
-        <Layout is404={true} footerContactUsId={id}>
-          <Helmet
-            title={`${site.siteMetadata.siteTitle} - Not Found`}
-            meta={[
-              {
-                name: 'description',
-                content: 'YLD - Engineering - Digital, NodeJS, React, AWS'
-              }
-            ]}
-          >
-            <html lang="en" />
-          </Helmet>
-          <Grid>
-            <Row>
-              <StyledCol width={[1]}>
-                <SectionTitle as="h1">{title}</SectionTitle>
-                <ReactMarkdown
-                  renderers={{
-                    // eslint-disable-next-line
-                    paragraph: props => <StyledBodyPrimary {...props} />
-                  }}
-                  source={copy}
-                />
-                <HomePageLink as={Link} to={ctaLink}>
-                  {ctaCopy}
-                </HomePageLink>
-              </StyledCol>
-            </Row>
-          </Grid>
-        </Layout>
+        <LogoStyleContext.Provider value="white">
+          <Layout is404={true} bgColor="blueBg" footerContactUsId={id}>
+            <Helmet
+              title={`${site.siteMetadata.siteTitle} - Not Found`}
+              meta={[
+                {
+                  name: 'description',
+                  content: 'YLD - Engineering - Digital, NodeJS, React, AWS'
+                }
+              ]}
+            >
+              <html lang="en" />
+            </Helmet>
+            <StyledBlueBackground>
+              <Grid>
+                <StyledRow>
+                  <TextCol>
+                    <TitleHeadline>{title}</TitleHeadline>
+                    <ReactMarkdown
+                      renderers={{
+                        paragraph: CopyText
```

Comment:
Is it not easier to just put `(...props) => <BodyPrimary {...props} reverse="true" />` here instead?

Reply:
I kinda like having all the text containers here together, and it avoids the lint error that was previously ignored :)